tox.ini

```diff
@@ -35,6 +36,15 @@ changedir = docs/source
 deps = -rdocs/requirements.txt
 commands = sphinx-build -W -b html -d {envtmpdir}/doctrees . {envtmpdir}/html
 
+[testenv:py38-qa]
+basepython = python3.8
+deps =
+    black
+    isort
+commands =
+    black -t py36 --check --diff auditlog auditlog_tests setup.py runtests.py
```

Comment:
Py35 is still in the test matrix?

Reply:
I changed it to `py35` in `project.toml` file.